dashboard-api/docker-compose-dev.yaml

```diff
@@ -0,0 +1,17 @@
+services:
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    environment:
+      STUDIESMANAGER_HOST: 0.0.0.0
```

Comment:
This is a bit strange to me, does using `0.0.0.0` mean that `STUDIESMANAGER` is running inside the same container as the `dashboard-api`?

Reply:
Yup this name is confusing, actually the STUDIESMANAGER is the DASHBOARD_API.

I changed the naming of the env variables to be more aligned with the names I used for the system service. 👌

Reply:
Ok, in that case. Do you think we should change the directory name and other occurrences of `dashboard-api` to match the env var name `studies-manager`? 

Having two different ways of naming the same thing could cause confusion. My preference is to be consistent with naming.